README.md

```diff
@@ -1 +1,3 @@
-# git101
\ No newline at end of file
+# git101
+
+Celia is cool
```

Comment:
Yeah sorry, I couldn't find an algorithm with a lower complexity than O(n^5). It's the sacrifice you have to make for such an important addition!